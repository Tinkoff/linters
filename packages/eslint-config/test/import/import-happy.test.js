import ESlint from 'eslint';
import path from 'path';

describe('import / happy path', () => {
  const cli = new ESlint.CLIEngine({
    cwd: path.join(__dirname, '..'),
    useEslintrc: false,
    baseConfig: {
      extends: ['../internal/import'],
      parserOptions: {
        sourceType: 'module',
      },
      env: {
        es6: true,
      },
    },
  });

  it('happy', () => {
    const codeframe = cli.getFormatter('codeframe');
    const report = cli.executeOnFiles([
      path.join(__dirname, './__fixtures__/import-happy.fixture.js'),
    ]);

    expect(codeframe(report.results)).toMatchSnapshot();
  });
});
