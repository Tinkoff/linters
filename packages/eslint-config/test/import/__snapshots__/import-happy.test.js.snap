// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`import / happy path happy 1`] = `""`;
